mpi-operator/Dockerfile

```diff
@@ -36,9 +36,9 @@ RUN sed -i "s/[ #]\(.*StrictHostKeyChecking \).*/ \1no/g" /etc/ssh/ssh_config \
     && sed -i "s/#\(Port \).*/\1$port/g" /etc/ssh/sshd_config
 
 # Install kubectl cli to be able to utilize file transfer functionality between containers
-ARG KUBECTL_VERSION=v1.28.2
+ARG KUBECTL_VERSION=v1.31.2
```

Comment:
Kubectl should be +/- 1 minor ver of k8s version (1.28.x)
I believe CNS wants to upgrade before Jan but idk if they'll make it.